remotespark/datawidgets/autovizwidget.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+import pandas as pd
+from ipywidgets import FlexBox
+from IPython.display import display
+
+from .encoding import Encoding
+from .encodingwidget import EncodingWidget
+from .ipywidgetfactory import IpyWidgetFactory
+
+
+class IpythonDisplay(object):
+    @staticmethod
+    def display_to_ipython(to_display):
+        display(to_display)
+
+
+class AutoVizWidgetTest(FlexBox):
```

Comment:
"""This class should not be used by anyone outside of the project. People should use AutoVizWidget instead.
This class is here for testing purposes."""

Some of the default arguments required are objects, which cannot be set as default arguments.